mint/run/core/healthcheck/healthcheck.go

```diff
@@ -144,34 +144,6 @@ func testReadinessEndpoint(endpoint string) {
 	defer successLogger(function, nil, startTime).Info()
 }
 
-func testPrometheusEndpoint(endpoint string) {
```

Comment:
This test is not useful since the endpoint is authenticated always. Will write a functional test once the mc PR and all the forecoming changes gets merged.